src/dotnet-svcutil/lib/Microsoft.Svcutil.NamedPipeMetadataImporter/Microsoft.Svcutil.NamedPipeMetadataImporter.csproj

```diff
@@ -1,14 +1,36 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyVersionFile>$(IntermediateOutputPath)\$(TargetFramework)\$(MSBuildProjectName).$(TargetFramework).version.cs</AssemblyVersionFile>
+  </PropertyGroup>
 	<PropertyGroup>
 		<TargetFrameworks>net6.0;net462</TargetFrameworks>
     <RootNamespace>Microsoft.Tools.ServiceModel.Svcutil</RootNamespace>
     <AssemblyName>$(MSBuildProjectName)</AssemblyName>
     <LangVersion>latest</LangVersion>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 	</PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="System.ServiceModel.NetNamedPipe" Version="6.0.*" Condition="'$(TargetFramework)' == 'net6.0'">
-			<PrivateAssets>contentfiles;analyzers;build</PrivateAssets>
+      <IncludeAssets>all</IncludeAssets>
 		</PackageReference>
     <Reference Include="System.ServiceModel" Condition="'$(TargetFramework)' == 'net462'" />
 	</ItemGroup>
+  <PropertyGroup>
+    <CustomOutputFolder>$([System.IO.Path]::GetFullPath('$(ArtifactsBinDir)\'))\dotnet-svcutil-lib\$(Configuration)\netstandard2.0</CustomOutputFolder>
+    <MajorVersion>8</MajorVersion>
```

Comment:
This part is needed only for branding, not related with the remove InternalAssets, right?  If so, does this belong to the other release PR you are preparing?

Reply:
Right, it's not related with removal of InternalAssets folder, it's also not necessary because NamedPipeMetadataImporter.csproj won't be packaged as package.  Removed.